presentation/frontend/src/presentation-frontend/Presentation.ts

```diff
@@ -92,7 +92,9 @@ export class Presentation {
       });
     }
     // eslint-disable-next-line @typescript-eslint/unbound-method
-    presentationManager.onNewiModelConnection = favoritePropertiesManager.initializeConnection.bind(favoritePropertiesManager);
+    presentationManager.startiModelInitialization = favoritePropertiesManager.startConnectionInitialization.bind(favoritePropertiesManager);
+    presentationManager.ensureiModelInitialized = favoritePropertiesManager.ensureInitialized.bind(favoritePropertiesManager);
```

Comment:
what we had previously wasn't great, let's change the calls to this:

```suggestion
    presentationManager.startiModelInitialization = (imodel) => favoritePropertiesManager.startConnectionInitialization(imodel);
    presentationManager.ensureiModelInitialized = (imodel) => favoritePropertiesManager.ensureInitialized(imodel);
```

Reply:
That's more clear, but an additional test had to be added for that to fix the coverage issue